changelog/fragments/1713257367-Remove-beat-state-metricset-from-elastic-agent-monitoring.yaml

```diff
@@ -11,12 +11,12 @@
 kind: bug-fix
 
 # Change summary; a 80ish characters long description of the change.
-summary: Removed beat state metricset from monitoring collect metrics are unused
+summary: Removed beat state metricset from monitoring metrics
 
 # Long description; in case the summary is not enough to describe the change
 # this field accommodate a description without length limits.
 # NOTE: This field will be rendered only for breaking-change and known-issue kinds at the moment.
-#description:
+description: Removing beat state metricsets reduces the document count of indices metrics-elastic_agent.*-* by 50%, saving around 13% disk space
```

Comment:
https://github.com/elastic/elastic-agent/issues/4153#issuecomment-2093194405

> Isn't the number of metrics produced dependent on the number of components running under agent? i.e. something like x document per beat per interval? so the % savings depends on the number of deployed integrations/managed beats?

We need to document which integrations we used to obtain this number and that that the savings depend on the number of inputs. More inputs = more monitoring data being written, so the total savings should be greater in terms of absolute numbers (MB) in that case.

Reply:
The numbers I got are obtained using only the system integration... Those should be replicated n times for any other component that belongs in the [supported component list](https://github.com/elastic/elastic-agent/blob/e7c4ac8bd7c5e1ae61fce9c97b373c64cb75559e/internal/pkg/agent/application/monitoring/v1_monitor.go#L67)

Reply:
Makes sense we drop the same amount of documents from every Beat